.travis.yml

```diff
@@ -1,16 +1,20 @@
+language: ruby
+script: "bundle exec rake test:units"
+sudo: false
```

Comment:
This enables the new infrastructure on Travis, which much faster build initialization times.